xep-0403.xml

```diff
@@ -61,32 +76,49 @@
     <li>The mechanism must work cleanly for participants with multiple clients.</li>
     <li>Standard presence messages must be used to share presence.</li>
     <li>Nick changes should be visible as changes (and not as a new user).</li>
-    <li>Where &xep0404; is not used,  participants must be able to directly contact other participants.</li>
+    <li>Where &xep0404; (MIX-ANON) is not used,  participants must be able to directly contact other participants.</li>
+    <li>IQ stanzas, including vCard, can be communicated through a channel.</li>
 
   </ol>
 
 </section1>
 
 <section1 topic='Concepts' anchor='concepts'>
+  
+ <section2 topic="Participant Information in Presence" anchor="concept-participant-info">
+   <p>
+     A MIX channel MAY distribute presence information about channel participants.   In order to share JID and Nick information about a participant, this information is encoded in the presence message.   This allows full presence information to be shared for each participant without the need for the client to perform any lookup.    
```

Comment:
Is this strictly OPTIONAL? I think if someone's implementing 403 it's actually mandatory isn't it?